pymc_marketing/clv/models/__init__.py

```diff
@@ -16,15 +16,18 @@
 
 from pymc_marketing.clv.models.basic import CLVModel
 from pymc_marketing.clv.models.beta_geo import BetaGeoModel
+from pymc_marketing.clv.models.beta_geo_beta_binom import BetaGeoBetaBinomModel
 from pymc_marketing.clv.models.gamma_gamma import (
     GammaGammaModel,
     GammaGammaModelIndividual,
 )
 from pymc_marketing.clv.models.pareto_nbd import ParetoNBDModel
 from pymc_marketing.clv.models.shifted_beta_geo import ShiftedBetaGeoModelIndividual
 
+
```

Comment:
Also removing this empty line (also get a complain locally)

Reply:
Fixed by https://github.com/pymc-labs/pymc-marketing/pull/1031/commits/b3abdab678ac7ae46a5491d26911cada469d494f using https://github.com/pymc-labs/pymc-marketing/pull/1031#issuecomment-2349538095